tests/test_process.py

```diff
@@ -1348,7 +1352,9 @@ def test_accented_directory(self):
         report_expected = (
             u"Name            Stmts   Miss  Cover\n"
             u"-----------------------------------\n"
-            u"\xe2%saccented.py       1      0   100%%\n" % os.sep
+            u"\xe2%saccented.py       1      0   100%%\n"
+            u"-----------------------------------\n"
+            u"TOTAL               1      0   100%%\n" % os.sep
```

Comment:
You don't need the string formatting and os.sep on this line.

Reply:
If I don't move the `% os.sep` piece to the last line (or after the trailing parenthesis) the code fails to execute with a syntax error. And the substitution has to be _somewhere_

I'll tweak this a bit and hopefully it's more obvious what's going on.